SweaterWeather/js/script.js

```diff
@@ -0,0 +1,116 @@
+var geoButton = document.getElementById('geo');
+var notifyButton = document.getElementById('notify');
+
+// Check onload status of geolocation permission.   
+navigator.permissions.query({name:'geolocation'}).then(function(result){ 
+ if (result.state == 'denied') {
+  geoButton.setAttribute('icon','communication:location-off');
+  geoButton.style.color = 'grey';
+ }
+ else if (result.state == 'granted') {
+  geoButton.setAttribute('icon','communication:location-on');
```

Comment:
You'll want this to be inside the DOMContentLoaded handler, since your geoButton lines of code won't necessarily work if the DOM hasn't loaded yet.